src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -440,102 +366,42 @@ public enum RC {
      */
     OK,
     /**
-     * Handler had some minor propblems, but it is recoverable, so parsing should keep going.
+     * Unrecoverable I/O error occured.
      */
-    WARN,
+    IO_ERROR,
     /**
-     * Unrecoverable error occured while handling option. No fall back, must quit.
+     * Unrecoverable error occured while handling option.
      */
-    QUIT
-  }
-
-  private static RC handleArgTty(Startup startup, Option opt) {
-    // TTY format parsing
-    final var ttyVal = opt.getValue();
-    final var fmts = ttyVal.split(",");
-    if (fmts.length > 0) {
-      // FIXME: why we support multiple TTY types in one invocation? fallback?
-      for (final var singleFmt : fmts) {
-        final var val = switch (singleFmt.trim()) {
-          case "table" -> TtyInterface.FORMAT_TABLE;
-          case "speed" -> TtyInterface.FORMAT_SPEED;
-          case "tty" -> TtyInterface.FORMAT_TTY;
-          case "halt" -> TtyInterface.FORMAT_HALT;
-          case "stats" -> TtyInterface.FORMAT_STATISTICS;
-          case "binary" -> TtyInterface.FORMAT_TABLE_BIN;
-          case "hex" -> TtyInterface.FORMAT_TABLE_HEX;
-          case "csv" -> TtyInterface.FORMAT_TABLE_CSV;
-          case "tabs" -> TtyInterface.FORMAT_TABLE_TABBED;
-          default -> 0;
-        };
-
-        if (val == 0) {
-          logger.error(S.get("ttyFormatError"));
-          return RC.QUIT;
-        }
-        startup.ttyFormat |= val;
-        return RC.OK;
-      }
-    }
-    logger.error(S.get("ttyFormatError"));
-    return RC.QUIT;
-  }
-
-  private static RC handleArgSubstitute(Startup startup, Option opt) {
-    final var fileA = new File(opt.getValues()[0]);
-    final var fileB = new File(opt.getValues()[1]);
-    if (!startup.substitutions.containsKey(fileA)) {
-      startup.substitutions.put(fileA, fileB);
-      return RC.OK;
-    }
-
-    // FIXME: warning should be sufficient here maybe?
-    logger.error(S.get("argDuplicateSubstitutionError"));
-    return RC.QUIT;
-  }
-
-  private static RC handleArgLoad(Startup startup, Option opt) {
-    if (startup.loadFile != null) {
-      logger.error(S.get("loadMultipleError"));
-      // FIXME: shouldn't we quit here? -> RC.QUIT;
-      return RC.WARN;
-    }
-    final var fileName = opt.getValue();
-    startup.loadFile = new File(fileName);
-    return RC.OK;
+    CLI_ERROR,
```

Comment:
I'd rename vague `CLI_ERROR` to `ARG_PARSING_ERROR` as this is what its real meaning is

Reply:
I'm happy to simplify to one exit code for all startup/parameter errors if there is consensus.